external-crates/move/crates/move-vm-types/src/views.rs

```diff
@@ -75,35 +74,35 @@ pub trait ValueView {
             }
 
             fn visit_vec_u8(&mut self, _depth: usize, vals: &[u8]) {
-                self.0 += ((size_of::<u8>() * vals.len()) as u64).into();
+                self.0 += (std::mem::size_of_val(vals) as u64).into();
```

Comment:
This looks good to me, but we should get an extra set of eyes just in case

Reply:
@tzakian

Reply:
This looks good to me -- checked out the docs/tried it out and looks good to me as well.